package.json

```diff
@@ -11,6 +11,7 @@
     "chalk": "^2.2.0",
     "closer-look": "git+ssh://git@github.com/art-institute-of-chicago/digital-labels-ui.git#master",
     "cross-spawn": "^6.0.5",
+    "custom-tour-builder": "git+https://github.com/art-institute-of-chicago/custom-tours.git#develop",
```

Comment:
Before we release, this should be changed to `main`.

Reply:
:+1: